### ISSUE TITLE
Display "Leere Seite" instead of "Kein Faksimile verfügbar"

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-web#132]
See https://faustedition.uni-wuerzburg.de/wiki/index.php/3._Februar_2016#Archiv.

Applies when the following conditions are met:
- `//page[.//docTranscript[not(@uri)] and .//comment()[contains(.,'leer')]]` **or**
- `//page[.//docTranscript[not(@uri)] and .//f:note/text()[contains(.,'Leer')]]`

**Comment by thvitt** [comment]
Parsing comments? Srsly? 

Requires implementation in `metadata2json.xsl` as well as in the document viewer.

**Comment by gerritbruening** [comment]
Well, if it were up to me, we maybe would have somethin else than these comments. But at least we have them ...

**Comment by thvitt** [comment]
Postponed until after #15.

The function that generates the label is at least three steps of data-mangling away from the actual information about the page in question, although the page empty information for every single page in the whole edition is available in a stupid global variable. The whole documentviewer js stuff needs proper data modelling and massive refactoring before it gets anywhere near to being maintainable.
